pygradle-plugin/build.gradle

```diff
@@ -10,9 +10,23 @@ plugins {
 apply from: "$rootDir/gradle/publishing.gradle"
 apply from: "$rootDir/gradle/code-quality.gradle"
 
-configurations {
-    testCompile.exclude group: 'org.codehaus.groovy', module: 'groovy-all'
-    integTestCompile.exclude group: 'org.codehaus.groovy', module: 'groovy-all'
+ext {
+    groovyVer = GroovySystem.version
+    spockGroovyVer = GroovySystem.version.replaceAll(/\.\d+$/, '')
+}
+
+//configurations {
+//    testCompile.exclude group: 'org.codehaus.groovy', module: 'groovy-all'
+//    integTestCompile.exclude group: 'org.codehaus.groovy', module: 'groovy-all'
+//    integTest2Compile.exclude group: 'org.codehaus.groovy', module: 'groovy-all'
+//}
+
+configurations.all {
+    //noinspection GroovyAssignabilityCheck
+    resolutionStrategy {
+        force "org.codehaus.groovy:groovy:${groovyVer}"
+        force "org.codehaus.groovy:groovy-all:${groovyVer}"
```

Comment:
I got into the habit of doing this because we always have problems with our plugins where some get upgraded before another.  This is how we fixed it.  But if you don't want it, ok.  What version of gradle?